src/test/CharterManager-unit.t.sol

```diff
@@ -218,17 +237,17 @@ contract CharterManagerTest is TestBase {
         );
     }
 
-    function init_ilk_ungate(uint256 Nib, uint256 Peace) public {
-        manager.file(ilk, "gate", 0);
-        manager.file(ilk, "Nib", Nib);
-        manager.file(ilk, "Peace", Peace);
+    function init_ilk_ungate(bytes32 ilk_, uint256 Nib, uint256 Peace) public {
+        manager.file(ilk_, "gate", 0);
+        manager.file(ilk_, "Nib", Nib);
+        manager.file(ilk_, "Peace", Peace);
     }
 
-    function init_ilk_gate(address user, uint256 nib, uint256 peace, uint256 uline) public {
-        manager.file(ilk, "gate", 1);
-        manager.file(ilk, user, "nib", nib);
-        manager.file(ilk, user, "peace", peace);
-        manager.file(ilk, user, "uline", uline);
+    function init_ilk_gate(bytes32 ilk_, address user, uint256 nib, uint256 peace, uint256 uline) public {
+        manager.file(ilk_, "gate", 1);
+        manager.file(ilk_, user, "nib", nib);
+        manager.file(ilk_, user, "peace", peace);
+        manager.file(ilk_, user, "uline", uline);
     }
 
     function init_user() internal returns (Usr a, Usr b) {
```

Comment:
Any reason to not remove this one and use the general one everywhere?

Reply:
I saw init_user() was used ~60 times so I didn't want to clutter the diff too much.
I guess I can do it as a last thing here once all comments are handled.

Reply:
Ahh ok makes sense.

Reply:
I'm giving my approval now.